### PR TITLE
[CARBONDATA-554] Fix check style

### PR DESCRIPTION
**Pull request by foryou2030**
# Why raise this pr?
It seems that the CI don't check style at present. when run command "mvn clean install -DskipTests", build failed

# How to solve it?
Fix check style

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/296/

File: processing/src/main/java/org/apache/carbondata/processing/newflow/sort/unsafe/sort/TimSort.java [hunk 1]
```diff
@@ -396,9 +407,9 @@ private SortState(Buffer a, Comparator<? super K> c, int len) {
        * computation below must be changed if MIN_MERGE is decreased.  See
        * the MIN_MERGE declaration above for more information.
        */
-      int stackLen = (len <    120  ?  5 :
-                      len <   1542  ? 10 :
-                      len < 119151  ? 19 : 40);
+      int stackLen = (len < 120 ? 5 :
+              len < 1542 ? 10 :
```
Comment (jackylk) [inline processing/src/main/java/org/apache/carbondata/processing/newflow/sort/unsafe/sort/TimSort.java]: incorrect indentation

**Comment by ravipesala** [comment]
Please don't fix the check style issues for this class as it is originally from Oracle Java and modified by Spark. I have kept in checkstyle xml to ignore the check style issues.

**Comment by foryou2030** [comment]
ok, closed. @ravipesala

**Pull request closed without merging**